Helper/Search/QueryBuilder.php

```diff
@@ -3,6 +3,7 @@
 namespace EMS\ClientHelperBundle\Helper\Search;
 
 use EMS\ClientHelperBundle\Helper\Elasticsearch\ClientRequest;
+use stdClass;
```

Comment:
stdClass is part of the SPL, use new \stdClass() in your code to prevent this use entry.